mllib/src/main/scala/org/apache/spark/mllib/evaluation/MulticlassMetrics.scala

```diff
@@ -138,24 +138,30 @@ class MulticlassMetrics @Since("1.1.0") (predictionAndLabels: RDD[(Double, Doubl
   /**
    * Returns precision
    */
-  @Since("1.1.0")
+  @deprecated("Use accuracy. This metric will be removed in 2.1.0.", "2.0.0")
   lazy val precision: Double = tpByClass.values.sum.toDouble / labelCount
 
   /**
    * Returns recall
-   * (equals to precision for multiclass classifier
+   * (equals to accuracy for multiclass classifier
    * because sum of all false positives is equal to sum
    * of all false negatives)
    */
   @Since("1.1.0")
-  lazy val recall: Double = precision
+  lazy val recall: Double = accuracy
+
+  /**
+   * Returns accuracy
```

Comment:
Just for complete obviousness, you might say that accuracy here is the total number of correctly classified instances out of the total number of instances. You can also move this after fMeasure since it forms a sort of (deprecated) group with precision and recall.